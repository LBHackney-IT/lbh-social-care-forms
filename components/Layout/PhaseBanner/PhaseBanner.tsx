interface Props {
  phase: string;
  faqLink: string;
  handbookLink: string;
}

const PhaseBanner = ({
  phase,
  faqLink,
  handbookLink,
}: Props): React.ReactElement => (
  <div className="govuk-phase-banner lbh-phase-banner lbh-container">
    <p className="govuk-phase-banner__content">
      <strong className="govuk-tag govuk-phase-banner__content__tag">
        {phase}
      </strong>
      <span className="govuk-phase-banner__text">
        Need help? Check out the{' '}
        <a className="govuk-link" href={faqLink}>
          FAQ
        </a>{' '}
        or the{' '}
        <a className="govuk-link" href={handbookLink}>
          handbook.
        </a>{' '}
        If you don&apos;t find the answer, contact{' '}
        <a href="mailto:social-care.support@hackney.gov.uk">support</a>{' '}
      </span>
    </p>
  </div>
);

export default PhaseBanner;
