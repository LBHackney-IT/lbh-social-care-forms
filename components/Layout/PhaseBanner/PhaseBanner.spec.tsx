import { render } from '@testing-library/react';

import PhaseBanner from './PhaseBanner';

describe('PhaseBanner', () => {
  it('should render properly', async () => {
    const { asFragment } = render(
      <PhaseBanner phase="alpha" faqLink="foo" handbookLink="bar" />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
