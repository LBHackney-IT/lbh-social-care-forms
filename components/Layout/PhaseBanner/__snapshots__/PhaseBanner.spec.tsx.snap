// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhaseBanner should render properly 1`] = `
<DocumentFragment>
  <div
    class="govuk-phase-banner lbh-phase-banner lbh-container"
  >
    <p
      class="govuk-phase-banner__content"
    >
      <strong
        class="govuk-tag govuk-phase-banner__content__tag"
      >
        alpha
      </strong>
      <span
        class="govuk-phase-banner__text"
      >
        Need help? Check out the 
        <a
          class="lbh-link lbh-link--no-visited-state"
          href="foo"
          rel="noreferrer noopener"
          target="_blank"
        >
          FAQ
        </a>
         or the 
        <a
          class="lbh-link lbh-link--no-visited-state"
          href="bar"
          rel="noreferrer noopener"
          target="_blank"
        >
          handbook
        </a>
        . If you don't find the answer, contact 
        <a
          href="mailto:social-care.support@hackney.gov.uk"
        >
          support
        </a>
         
      </span>
    </p>
  </div>
</DocumentFragment>
`;
